lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -68,6 +68,15 @@ class DetailsRenderer {
       case 'table':
         // @ts-ignore - TODO(bckenny): Fix type hierarchy
         return this._renderTable(/** @type {TableDetailsJSON} */ (details));
+      case 'list':
+        return this._renderList(
+          // @ts-ignore - TODO(bckenny): Fix type hierarchy
+          /** @type {LH.Audit.Details.List} */ (details)
+        );
+      case 'snippet':
```

Comment:
let's not add `snippet` here (it's just going to be immediately removed anyways)